tests/scripts/save_dataset_to_safetensors.py

```diff
@@ -23,7 +23,7 @@
 doesnt need to be merged into the `main` branch. Then you need to run this script and update the tests artifacts.
 
 Example usage:
-    `python tests/scripts/save_dataset_to_safetensors.py`
+    `DATA_DIR=tests/data python tests/scripts/save_dataset_to_safetensors.py`
```

Comment:
It is supposed to be used without. We should revert.
```suggestion
    python tests/scripts/save_dataset_to_safetensors.py`
```

Reply:
Got it